src/utils/upload/index.ts

```diff
@@ -74,6 +77,13 @@ export function getUploadLogs() {
         .map((log) => `[${formatDateTime(log.timestamp)}] ${log.logLine}`);
 }
 
-export function getFileNameSize(file: File) {
+export function getFileNameSize(file: File | ElectronFile) {
     return `${file.name}_${convertBytesToHumanReadable(file.size)}`;
 }
+
+export function areFileWithCollectionsSame(
+    firstFile: FileWithCollection,
+    secondFile: FileWithCollection
+): boolean {
+    return firstFile.localID === secondFile.localID;
```

Comment:
Sorry, i haven't read the complete code. 
What exactly is this `local_ID` and how does it get set? Say if I upload a desktop folder for X files. And then I delete few files from that folder and try to upload again, will the localID change?

Reply:
local_ID is just a sequential ID given to the files queued for upload, it's only used in the context of an upload session to refer to those files.

yes on a new upload, all the files will be assigned a new local_ID, but we don't persist the old_ids anywhere so it won't be a problem